bucket/buildcache.json

```
@@ -0,0 +1,37 @@
+{
+    "version": "0.28.4",
+    "description": "BuildCache is an advanced compiler accelerator that caches and reuses build results to avoid unnecessary re-compilations, and thereby speeding up the build process.",
+    "homepage": "https://github.com/mbitsnbites/buildcache",
+    "license": "Zlb",
```

```suggestion
    "license": "Zlib",
```

correct short identifier